docs/deploy/configurations.md

```diff
@@ -102,6 +102,12 @@ Currently, you can set only 1 namespace to watch in this flag. See [this Kuberne
 |webhook-cert-file                      | string                          | tls.crt | The server certificate name |
 |webhook-key-file                       | string                          | tls.key | The server key name |
 
+|Feature-gate Supported Key             | Type                            | Default Value   | Description |
```

Comment:
This table should be on a separate section, something like
### Controller Feature Gates

And provide description on how to specify via the command like flag